fvtest/porttest/si.cpp

```diff
@@ -867,7 +867,10 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_CORE_FILE)
 		uint64_t originalHardLimit = 0;
 		uint64_t currentLimit = 0;
 		const uint64_t descriptorLimit = 256;
-
+	#if defined(OSX)
```

Comment:
Preprocessor definitions should be at column zero, IE don't indent this line, the `#else`, or the `#endif`.

Reply:
This `#if` block should be moved above the local variable declarations, to the very front of the test body.